index.bs

```diff
@@ -5930,6 +5930,17 @@ H(z) = \frac{\frac{b_0}{a_0} + \frac{b_1}{a_0}z^{-1} + \frac{b_2}{a_0}z^{-2}}
 $$
 </pre>
 
+This <a href="https://en.wikipedia.org/wiki/Infinite_impulse_response#Transfer_function_derivation">
+is equivalent</a> to the following time-domain equation:
+
+<pre nohighlight>
+$$
+y(n) = \frac{b_0}{a_0} x(n) + \frac{b_1}{a_0} x(n - 1) + \frac{b_2}{a_0} x(n - 2)
+         - \frac{a_1}{a_0} y(n - 1) - \frac{a_2}{a_0} y(n - 2)
+$$
```

Comment:
For consistency with the IIR filter, I'd write

a0*y(n)+a1*y(n-1)+a2*y(n-2) = b0*x(n)+b1*x(n-1)*b2*x(n-2)